### PR TITLE
Add fr_CA langages files

### PR DESCRIPTION
**Pull request by ghost**
The files are based on fr_FR with improvement after linguistic revision from www.banq.qc.ca internal linguistic service.

As discussed.
- This replace a previous pull request that was partly lost by fork deletion (https://github.com/TGMPA/TGM-Plugin-Activation/pull/730)
- Only contains changes for fr_CA.

### CODE REVIEW

**Comment by ghost** [comment]
I was not able to pinpoint very problematic case, it's really up to fr_FR to give it a look and see if they could improve their version using our changes.

I feel confident that this can be used as fr_CA base and I can give it a second look if someone else want later to modify it or improve it.

File: languages/tgmpa-fr_CA.po [hunk 1]
```diff
@@ -0,0 +1,491 @@
+# Translation of TGM Plugin Activation in French (France)
```
Comment (jrfnl) [inline languages/tgmpa-fr_CA.po]: This header will need to be manually edited to read `French (Canada)`

**Review by jrfnl — COMMENTED** [review]
Hi Eric, sorry for the trouble with the deleted fork and thanks for recovering the translation!

I only have one small remark, other than that, I'm happy to merge this in.

**Comment by ghost** [comment]
Oh sorry, I should have noticed. Fixed.

I have sent a email to person that last updated the fr_FR.

**Review by jrfnl — APPROVED** [review]

**Comment by ghost** [comment]
Thanks!

**Comment by jrfnl** [comment]
@eric-gagnon No worries & thanks for the quick fix. Thanks for your contribution. The translation will be included in the next release.

**Pull request merged**